src/System.Globalization/tests/CultureInfo/CultureInfoTests.netstandard1.7.cs

```diff
@@ -75,26 +75,35 @@ public void LcidTest(string cultureName, int lcid, string specificCultureName, s
         }
 
         [Fact]
-        [ActiveIssue(11609, TestPlatforms.AnyUnix)]
         public void InstalledUICultureTest()
         {
             // as we didn't change current UI culture, Installed UI culture should match current UI culture
             Assert.Equal(CultureInfo.CurrentUICulture, CultureInfo.InstalledUICulture);
         }
-
-        [Fact]
-        [ActiveIssue(11609, TestPlatforms.AnyUnix)]
-        public void GetCulturesTest()
+        
+        [Theory]
+        [MemberData(nameof(CultureInfo_TestData))]
+        public void GetCulturesTest(string cultureName, int lcid, string specificCultureName, string threeLetterISOLanguageName, string threeLetterWindowsLanguageName, string alternativeCultureName)
         {
+            bool found = false;
             foreach (CultureInfo ci in CultureInfo.GetCultures(CultureTypes.NeutralCultures))
+            {
                 Assert.True(ci.IsNeutralCulture || ci.Equals(CultureInfo.InvariantCulture), "Expected Neutral Cultures or invariant");
+                if (!found && (ci.Name.Equals(cultureName, StringComparison.OrdinalIgnoreCase) || ci.Name.Equals(alternativeCultureName, StringComparison.OrdinalIgnoreCase)))
+                    found = true;
+            }
```

Comment:
I think this might be better as two separate things:
```C#
Assert.All(CultureInfo.GetCultures(CultureTypes.NeutralCultures), c => Assert.True(c.IsNeutralCulture || c.Equals(CultureInfo.InvariantCulture));
```
so that if it fails, we know which culture caused it to fail, and then:
```C#
bool found = CultureInfo.GetCultures(CultureTypes.NeutralCulture).Any(c => c.Name.Equals(cultureName, SC.OIC) || c.Name.Equals(alternativeCultureName, SC.OIC));
```